tests/test_exceptions.cpp

```diff
@@ -105,11 +105,6 @@ struct PythonAlreadySetInDestructor {
     py::str s;
 };
 
-std::string error_already_set_what(const py::object &exc_type, const py::object &exc_value) {
-    PyErr_SetObject(exc_type.ptr(), exc_value.ptr());
-    return py::error_already_set().what();
-}
```

Comment:
Was this just not being used?

Reply:
Ahh, "Removing a stray block of code", got it.

Reply:
Yeah, I just forgot to remove that block after moving the code into a lambda function.